scripting/get5.sp

```diff
@@ -963,7 +968,7 @@ public Action Event_MatchOver(Event event, const char[] name, bool dontBroadcast
       g_MapChangePending = true;
       Get5_MessageToAll("%t", "NextSeriesMapInfoMessage", nextMap);
       ChangeState(Get5State_PostGame);
-      CreateTimer(minDelay, Timer_NextMatchMap);
+      CreateTimer(0.1, Timer_NextMatchMap);
```

Comment:
Unfortunately my suggestion had an issue. This won't work. Triggering Timer_NextMatchMap too early will cause the demo to stop too early, so the delay probably does have to be added in two places. Sorry for the bad suggestion.